src/ligero/utils.rs

```diff
@@ -229,7 +229,7 @@ pub(crate) fn calculate_t<F: PrimeField>(
     let residual = codeword_len / 2.0_f64.powi(field_bits);
     let rhs = (2.0_f64.powi(-sec_param) - residual).log2();
     if !(rhs.is_normal()) {
-        return Err(Error::InvalidSecurityGuarantee);
+        return Err(Error::InvalidParameters(format!("There are not enough bits in the field to support the requested security parameter. The field has {} bits, but the requested security parameter is {}.", field_bits, sec_param)));
```

Comment:
Let's remove this part `The field has {} bits, but the requested security parameter is {}.", field_bits, sec_param)` or let's add the `codeword_len` too. Since it is not enough to only have a field bigger than the security parameter.